<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>405</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>709</width>
    <height>405</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>709</width>
    <height>405</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ProgramShutdown</string>
  </property>
  <property name="windowIcon">
   <iconset resource="application.qrc">
    <normaloff>:/switch_off.png</normaloff>:/switch_off.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>261</width>
          <height>32</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>110</y>
          <width>51</width>
          <height>31</height>
         </rect>
        </property>
        <property name="maximum">
         <number>24</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_2">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>110</y>
          <width>51</width>
          <height>31</height>
         </rect>
        </property>
        <property name="maximum">
         <number>59</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_3">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>110</y>
          <width>51</width>
          <height>31</height>
         </rect>
        </property>
        <property name="maximum">
         <number>59</number>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>110</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Shut out after:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_4">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>110</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="maximum">
         <number>31</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>110</y>
          <width>51</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>days</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="progressBar">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>210</y>
          <width>481</width>
          <height>31</height>
         </rect>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="value">
         <number>10000</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>210</y>
          <width>151</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Remaining time:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>30</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>30</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Refresh</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>80</y>
          <width>149</width>
          <height>149</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>40</y>
          <width>251</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>ProgramShutdown</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>80</y>
          <width>61</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>v0.1</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>210</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>by GhostTom</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>230</y>
          <width>211</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>in Ivano-Frankivsk, Ukraine</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>260</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>on 20-04-2016</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>709</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="application.qrc"/>
 </resources>
 <connections/>
</ui>
